### PR TITLE
feat(MangaFreak): add presence

### LINKED ISSUE #5255: MangaFreak | mangafreak.net
### Discussed in https://github.com/PreMiD/Presences/discussions/4867

<div type='discussions-op-text'>

<sup>Originally posted by **javereg** November  4, 2021</sup>
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

Discord name: Javere#3944

**What's the name of the service?** https://w12.mangafreak.net/ | MangaFreak

**What should the Presence display?** 
-Manga title
-Chapter number
-Time spent reading that chapter
-Browsing thru the webpage

**If possible, please provide a logo for the service 
7JHOJADN_400x400

</div>

### PR DESCRIPTION
**Pull request by RisingSunLight42**
## Description 

Resolves #5255 by adding the requested presence.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

mangafreak_proof_7

mangafreak_proof_6

mangafreak_proof_5

mangafreak_proof_4

mangafreak_proof_3

mangafreak_proof_2

mangafreak_proof_1

</details>

### CODE REVIEW

**Review by EncryptedDev — APPROVED** [review]

**Comment by RisingSunLight42** [comment]
I've just realized that logo and thumbnail are the same, I'll change that when I can.

File: websites/M/MangaFreak/presence.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
+const presence = new Presence({ clientId: "1007662369058594937" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	logo = "https://i.imgur.com/1yBi5L5.jpg",
+	searchingImg = "https://i.imgur.com/OIgfjTG.png",
+	readingImg = "https://i.imgur.com/53N4eY6.png";
```
Comment (Bas950) [inline websites/M/MangaFreak/presence.ts:5]: enum would be nice

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/MangaFreak/presence.ts (hunk 1 above)
Comment (RisingSunLight42) [reply websites/M/MangaFreak/presence.ts:5]: Yeah! I didn't know about that, I will change that when I can

Comment (RisingSunLight42) [reply websites/M/MangaFreak/presence.ts]: Resolved in https://github.com/PreMiD/Presences/pull/6650/commits/8c708eda25c97330152a136eec3852576ba8edb5

**Comment by RisingSunLight42** [comment]
> I've just realized that logo and thumbnail are the same, I'll change that when I can.

Fixed in https://github.com/PreMiD/Presences/pull/6650/commits/8c708eda25c97330152a136eec3852576ba8edb5

**Review by Bas950 — APPROVED** [review]

**Pull request merged**